Lombiq.Tests.UI/Models/TestDumpItemGeneric.cs

```diff
@@ -4,21 +4,21 @@
 
 namespace Lombiq.Tests.UI.Models;
 
-public class FailureDumpItemGeneric<TContent> : IFailureDumpItem
+public class TestDumpItemGeneric<TContent> : ITestDumpItem
 {
     private readonly TContent _content;
     private readonly Func<TContent, Task<Stream>> _getStream;
     private readonly Action<TContent> _dispose;
     private bool _disposed;
 
-    public FailureDumpItemGeneric(
+    public TestDumpItemGeneric(
         TContent content,
         Func<TContent, Task<Stream>> getStream = null,
         Action<TContent> dispose = null)
     {
         if (content is not Stream && getStream == null)
         {
-            throw new ArgumentException($"{nameof(content)} is not a Stream, {nameof(getStream)} can't be null");
+            throw new ArgumentException($"{nameof(content)} is not a Stream and {nameof(getStream)} can't be null.");
```

Comment:
It would be better to phrase the exception message as an instruction than a status report. Easier to understand during debugging,

```suggestion
            throw new ArgumentException(
                $"{nameof(content)} must be of type {nameof(Stream)} or {nameof(getStream)} must not be null.");
```

Reply:
Yep.